### ISSUE TITLE
I can't run script

### ISSUE THREAD

**Issue by LeGamerDu92** [MainSilent/Discord-Screenshare#56]
When I try to run that script `node main.js` it crashes with that error 
`Webdriver started
/home/pi/Discord-Screenshare-master/node_modules/selenium-webdriver/remote/index.js:248
                reject(Error(e.message))
                       ^

Error: Server terminated early with status 2
    at /home/pi/Discord-Screenshare-master/node_modules/selenium-webdriver/remote/index.js:248:24
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.3.0
`
Please help, what can I do to repair that ?

**Comment by LeGamerDu92** [comment]
I used the docker file but it exited error 1

**Comment by LeGamerDu92** [comment]
Raspberry Pi OS arm64

**Comment by LeGamerDu92** [comment]
does docker works on windows ?

**Comment by MainSilent** [comment]
> Raspberry Pi OS arm64

I don't believe it would work on ARM since many people tried using android and it failed.

**Comment by LeGamerDu92** [comment]
oh..

**Comment by LeGamerDu92** [comment]
and do I have the good version of nodejs ?

**Comment by MainSilent** [comment]
> and do I have the good version of nodejs ?

I only tried on 17.9.0 you can use nodeenv.

**Comment by MainSilent** [comment]
You need to have chrome in order to use the chromedriver.

**Comment by MainSilent** [comment]
Sorry, I misunderstood your last comment, You need to match the chromium version with the chrome driver.

**Comment by MainSilent** [comment]
https://chromedriver.chromium.org/downloads

**Comment by LeGamerDu92** [comment]
Thanks ! (i will close that ticket if it works)

**Comment by LeGamerDu92** [comment]
I have now a file named chromedriver(zip extracted), what can i do with it ?

**Comment by LeGamerDu92** [comment]
Where can i place that file ?

**Comment by LeGamerDu92** [comment]
The answer of that question is : No, ARM is not supported
